var/spack/repos/builtin/packages/hpe-mpi/package.py

```diff
@@ -6,12 +33,35 @@ class HpeMpi(Package):
     homepage = "http://www.example.com"
     url      = "http://www.example.com/hpempi-1.0.tar.gz"
 
-    version('2.16', '0123456789abcdef0123456789abcdef')
+    version('2.21', '2dd6c53a82993c4df929fdf898ac3d19',
+        url='file:///gpfs/bbp.cscs.ch/apps/hpc/download/hpe-mpi/hpe-mpi-2.21.tar.xz')
 
     provides('mpi')
 
-    def do_fetch(self, mirror_only=False):
-        raise RuntimeError('HPE MPI should be configured as external package')
+
+    @run_before('install')
+    def unpack(self):
+        rpm2cpio = spack.util.executable.which('rpm2cpio')
+        cpio = spack.util.executable.which('cpio')
+
+        print(self.stage)
```

Comment:
to remove

Reply:
You mean the print, right? indeed that got forgotten...